### PR TITLE
ACC-111 deal with ufw, and set correct onlyoffice.documentserver.host

### PR DESCRIPTION
**Pull request by madhifallah**
After a first deployment on acceptance, it appeaed that I need to deal with UFW and set a correct value for onlyoffice.documentserver.host

### CODE REVIEW

File: etc/plf/setenv-customize.sh [hunk 1]
```diff
@@ -64,7 +64,7 @@ fi
 
 # eXo Onlyoffice addon
 if ${DEPLOYMENT_ONLYOFFICE_DOCUMENTSERVER_ENABLED}; then
-    CATALINA_OPTS="${CATALINA_OPTS} -Donlyoffice.documentserver.host=localhost:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}"
+    CATALINA_OPTS="${CATALINA_OPTS} -Donlyoffice.documentserver.host=${DEPLOYMENT_APACHE_VHOST_ALIAS}:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}"
```
Comment (vsellier) [inline etc/plf/setenv-customize.sh]: What will happen when there will be no vhost declared ?
Why not using DEPLOYMENT_URL ?  https://github.com/exoplatform/adt/blob/master/_functions.sh#L1144

File: etc/plf/standalone-customize.conf [hunk 2]
```diff
@@ -62,7 +62,7 @@ if ${DEPLOYMENT_CHAT_ENABLED}; then
 fi
 # eXo Onlyoffice addon
 if ${DEPLOYMENT_ONLYOFFICE_DOCUMENTSERVER_ENABLED}; then
-    CATALINA_OPTS="${CATALINA_OPTS} -Donlyoffice.documentserver.host=localhost:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}"
+    CATALINA_OPTS="${CATALINA_OPTS} -Donlyoffice.documentserver.host=${DEPLOYMENT_APACHE_VHOST_ALIAS}:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}"
```
Comment (vsellier) [inline etc/plf/standalone-customize.conf]: Same remark as for tomcat

File: etc/plf/setenv-customize.sh (hunk 1 above)
Comment (vsellier) [reply etc/plf/setenv-customize.sh]: ok it's the url not the host but IMO you should introduce something similar

Comment (madhifallah) [reply etc/plf/setenv-customize.sh]: I totally agree

Comment (madhifallah) [reply etc/plf/setenv-customize.sh]: What do you think of introducing a new var : 
env_var  "ONLYOFFICE_EXT_HOST_PORT" $(echo ${DEPLOYMENT_URL} | sed 's/https\?:\/\///')

?

**Pull request merged**